db/schema.rb

```diff
@@ -10,100 +10,103 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20181113195810) do
+ActiveRecord::Schema.define(version: 20190612161235) do
+
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
```

Comment:
Why is this here? In general, why does this file have that many changes?

Reply:
I think some unwanted changes were introduced during db migrations . I will check it again in my new PR against the GSoC19 branch.